docs/cue.md

```diff
@@ -60,5 +78,92 @@ it should contain:
 - under `spec.plugin` :
   - the query's `kind`.
   - the query's `spec` containing:
-		- a `datasource` field that holds the `kind` of datasource corresponding to this query type.
-		- any other field you want for this query plugin.
+    - a `datasource` field that holds the `kind` of datasource corresponding to this query type.
+    - any other field you want for this query plugin.
+
+# Migration from Grafana
+
+A Perses plugin can optionally embed a `mig.cuepart`* file that is basically describing in CUE language how to convert a given Grafana object into an instance of this plugin. In such case your plugin is considered as the Perses equivalent of this Grafana object type, and it will be used as part of the translation process when a Grafana dashboard is received on the `/migrate` endpoint.
+
+*\* this is mandatory to have it named that way. We put in place this constraint because it makes sense to have a single file containing the remapping logic, with the benefit of making the backend logic easier (no need to search for the file). It's also easier to check the migration logic of the different plugins, because you know which file to look for.* 
+*\* the .cuepart extension is a homemade one. It is used for CUE files that may contain some placeholders for string replacements, that are not valid CUE syntax.*
```

Comment:
Probably you need to add an empty line, otherwise Markdown is considering it's the same paragraph and it will concat both on the same line.